virt/virt-2-5-release-notes.adoc

```diff
@@ -106,6 +109,7 @@ The *Pending Changes* banner at the top of the page displays a list of all chang
 //Placeholder for new content.
 
 //CNV-7293 - Use OLM stable channel for installation
+* There is a new *Update Channel* called *stable*. You must select the *stable* option during installation or upgrade so that {product-title} automatically consumes the correct version of {VirtProductName}.
```

Comment:
Didn't realize that I included a placeholder for this story in two places. Maybe just have it here in the notable technical changes section?